tests/unit-tests/Commands/FreshCommandTest.php

```diff
@@ -63,6 +63,37 @@ public function runs_fresh_with_seeding_on_tenants()
         ]);
     }
 
+    /**
+     * @test
+     */
+    public function purges_connection_after_running_fresh_on_multiple_tenants()
+    {
+        $websiteB = new Website();
```

Comment:
```suggestion
        $website = new Website();
```

`$website` would be unique in this scope too. As such there's no reason to name this object differently.

Reply:
Done.